src/app/services/api.ts

```diff
@@ -25,6 +25,8 @@ export const api = createApi({
         'Challenges',
         'Idle_Members',
         'Progress_Details',
+        'User_Standup',
+        'TASK_REQUEST',
```

Comment:
why are we not following the one-name convention?

Reply:
I am not aware of any such convention and I referred the `Mine_Tasks` key for this